sources/packages/api/src/services/disbursement-schedule-service/disbursement-schedule-service.ts

```diff
@@ -236,4 +235,20 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
   ): Promise<DisbursementSchedule> {
     return this.repo.findOne({ documentNumber: documentNumber });
   }
+
+  /**
+   * Update DisbursementSchedule with documentNumber
+   * @param documentNumber document Number
+   * @returns the result of update.
+   */
+  async updateDisbursementScheduleDocumentNumber(
+    applicationId: number,
+  ): Promise<DisbursementSchedule> {
+    const disbursementScheduleToUpdate = await this.repo.findOne({
```

Comment:
If I am not wrong, an application can have one or two disbursement schedules associate with it. How do we know that we are updating the expected one here?

Reply:
The way that the query was changed does not change the fact that the update can happen in two records because these two records will be associated with the same application.
I will not consider this as a blocker because there will be a second feature to deal with the second COE.